orchagent/orch.h

```diff
@@ -33,6 +33,7 @@ const char config_db_key_delimiter = '|';
 
 #define MLNX_PLATFORM_SUBSTRING "mellanox"
 #define BRCM_PLATFORM_SUBSTRING "broadcom"
+#define BFN_PLATFORM_SUBSTRING  "barefoot"
```

Comment:
add BFN_PLATFORM_SUBSTRING "vs" here.

I will export the env in the vs.

Reply:
Will do. Thanks!

Reply:
check here. 

https://github.com/Azure/sonic-buildimage/pull/1860

Reply:
I mean #define VS_PLATFORM_SUBSTRING "vs"

Reply:
Fixed